spring5/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>jdbi3-parent</artifactId>
+        <groupId>org.jdbi</groupId>
+        <version>3.6.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jdbi3-spring5</artifactId>
+    <name>jdbi3 Spring 5</name>
+    <description>jdbi Spring 4/5 forward compatibiility testing</description>
+    <url>http://jdbi.org/</url>
+
+    <properties>
+        <dep.spring.version>5.1.3.RELEASE</dep.spring.version>
```

Comment:
Does this actually succeed in changing the dependency version?

Reply:
according to intellij > maven > show effective pom, it does